pkg/exchange/coinbase/api/v1/get_order_trades_request.go

```diff
@@ -0,0 +1,46 @@
+package coinbase
+
+import (
+	"time"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/requestgen"
+)
+
+type Trade struct {
+	TradeID         int              `json:"trade_id"`
+	ProductID       string           `json:"product_id"`
+	OrderID         string           `json:"order_id"`
+	UserID          string           `json:"user_id"`
+	ProfileID       string           `json:"profile_id"`
+	Liquidity       Liquidity        `json:"liquidity"`
+	Price           fixedpoint.Value `json:"price"`
+	Size            fixedpoint.Value `json:"size"`
+	Fee             fixedpoint.Value `json:"fee"`
+	CreatedAt       time.Time        `json:"created_at"`
+	Side            SideType         `json:"side"`
+	Settled         bool             `json:"settled"`
+	UsdVolume       string           `json:"usd_volume"`
+	FundingCurrency string           `json:"funding_currency"`
+}
+
+type TradeSnapshot []Trade
+
+//go:generate requestgen -method GET -url "/orders/fills" -type GetOrderTradesRequest -responseType .TradeSnapshot
+type GetOrderTradesRequest struct {
+	client requestgen.AuthenticatedAPIClient
+
+	orderID    string      `param:"order_id,required"`
+	limit      int         `param:"limit"`
+	before     *string     `param:"before"`
+	after      *string     `param:"after"`
```

Comment:
you can use *time.Time here?

like this https://github.com/c9s/bbgo/blob/main/pkg/exchange/max/maxapi/get_k_lines_request.go#L22